### ISSUE TITLE
Expose svd_flip for controlling sign indeterminacy

### ISSUE THREAD

**Issue by eric-czech** [dask/dask#6599]
Building on https://github.com/dask/dask/pull/6591#pullrequestreview-481806234, it would be helpful if there was a scalable solution for normalizing eigenvector signs much like [sklearn.svd_flip](https://github.com/scikit-learn/scikit-learn/blob/7bbd0d56530ad3758841595d81ab8deed069879d/sklearn/utils/extmath.py#L504).  

There is some discussion at https://github.com/dask/dask/pull/6591#issuecomment-686487526 on where it could live and how it would be applied.

This is particularly useful for library maintainers that want to compare results from different SVD implementations, though it is also necessary for users that wish to make SVD output reproducible.

- [ ] Replace the svd_flip function used in the unit tests for https://github.com/dask/dask/pull/6591

**Comment by eric-czech** [comment]
Done in https://github.com/dask/dask/pull/6616
